### ISSUE TITLE
Sorting in partner goals

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1795]
**Is your feature request related to a problem? Please describe.**
It's very annoying when you want to rearrange goals in partner view - but they reset the next time you visit.

**Describe the solution you'd like**
Allow sorting in partner goals.

**Describe alternatives you've considered**
Follow sorting order of the partner. This is complicated and impossible when sharing between multiple people.

**Additional context**
None.

**Comment by tijlleenders** [comment]
@Tushar-4781 What is your opinion on this?

**Comment by tijlleenders** [comment]
@Tushar-4781 *bump*

**Comment by Tushar-4781** [comment]
@tijlleenders  
yeah, user priorities matter. we can copy the implementation from our goals side

**Comment by tijlleenders** [comment]
Blocked by #1918
